src/components/componentGroup.ts

```diff
@@ -23,6 +23,48 @@ export module Component {
       components.forEach((c: AbstractComponent) => this._addComponent(c));
     }
 
+    /**
+     * Retrieves closest PlotData to queryPoint across all plots in this group.
+     * Plots define their own notion of closeness; this function compares results across
```

Comment:
How does the function compare results / what are the results you are referring to?